mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -3,32 +3,101 @@
 import java.util.Arrays;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+
+import static org.arquillian.smart.testing.mvn.ext.ApplicablePlugins.FAILSAFE;
+import static org.arquillian.smart.testing.mvn.ext.ApplicablePlugins.SUREFIRE;
 
 class MavenPropertyResolver {
```

Comment:
Having this whole change of the class `MavenPropertyResolver`, then it would make sense to move all the static methods to the `SkipInstallationChecker` and make then non-static. Then you would have this class separated only for methods operating with the `Model` and plugins. Also the name of the class could change then, for example to `SkipModuleChecker`